substrate/frame/support/src/lib.rs

```diff
@@ -915,7 +915,7 @@ pub mod pallet_prelude {
 	pub use scale_info::TypeInfo;
 	pub use sp_inherents::MakeFatalError;
 	pub use sp_runtime::{
-		traits::{MaybeSerializeDeserialize, Member, ValidateUnsigned},
+		traits::{CheckedAdd, MaybeSerializeDeserialize, Member, One, ValidateUnsigned},
```

Comment:
What about all the other `Checked*` traits and `Zero`?

Reply:
Added them.